client/my-sites/themes/collections/showcase-theme-collection.tsx

```diff
@@ -26,8 +29,36 @@ export default function ShowcaseThemeCollection( {
 	query,
 	title,
 	onSeeAll,
+	collectionIndex,
 }: ShowcaseThemeCollectionProps ): ReactElement {
-	const { getPrice, themes, isActive, isInstalling, siteId } = useThemeCollection( query );
+	const { getPrice, themes, isActive, isInstalling, siteId, getThemeType } =
+		useThemeCollection( query );
+
+	const { recordThemeClick, recordThemeStyleVariationClick, recordThemesStyleVariationMoreClick } =
+		getThemeShowcaseEventRecorder(
+			query,
+			themes,
+			getThemeType,
+			isActive,
+			collectionSlug,
+			collectionIndex
+		);
+
+	const onScreenshotClick = ( themeId: string, resultsRank: number ) => {
+		trackClick( 'theme', 'screenshot' );
+		recordThemeClick( themeId, resultsRank, 'screenshot_info' );
+	};
+
+	const onStyleVariationClick = (
+		themeId: string,
+		resultsRank: number,
+		variation: { slug: string }
+	) => {
+		recordThemeClick( themeId, resultsRank, 'style_variation', variation?.slug );
+		variation
+			? recordThemeStyleVariationClick( themeId, resultsRank, '', variation.slug )
+			: recordThemesStyleVariationMoreClick( themeId, resultsRank );
```

Comment:
These are the same events we're firing on the Theme Showcase. I think that it doesn't make much sense if at all to have them. For starters, we fire a `calypso_themeshowcase_theme_click` with whatever is going on. And then we fire another event with basically the same information but a different name.

I think we could simplify this by just changing the action of the main event to either `style_variation` or `more_style_variations` and we would have the same result but we could reduce the amount of code needed for this.

I check if the events are being used in funnels and the `calypso_themeshowcase_theme_style_variation_more_click` isn't used at all and the `calypso_themeshowcase_theme_style_variation_click` is only used in one funnel. So I think it's safe to delete these events.

Reply:
According to Tracks, both these events are owned by Lego. I'd check with them first. 🙂 

(I'd also definitely address this in a follow-up, as I've already rambled at length in another comment: https://github.com/Automattic/wp-calypso/pull/83456/files/7804c4b3885bf1f663d360f07309f86e0c983d81#r1372986824 😄 )

Reply:
Let's continue the conversation in that comment then. Resolving this comment.